zhaquirks/tuya/ts000f_switch.py

```diff
@@ -0,0 +1,91 @@
+"""tuya TS000F xkap8wtb On/Off In-line Switches with metering support."""
+
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import (
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODEL,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import (
+    TuyaZBE000Cluster,
+    TuyaZBElectricalMeasurement,
+    TuyaZBMeteringCluster,
+    TuyaZBOnOffAttributeCluster,
+)
+from zhaquirks.tuya.mcu import EnchantedDevice
+
+
+class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+    """Tuya 1 gang wall switch with metering and restore power state support."""
+
+    signature = {
+        MODEL: "TS000F",
+        ENDPOINTS: {
+            # SizePrefixedSimpleDescriptor(endpoint=242, profile=41440, device_type=97, device_version=0, input_clusters=[], output_clusters=[33])
+            # input_clusters=["0x0000","0x0003","0x0004","0x0005","0x0006", "0x000a", "0x0702","0x0b04","0x1000","0x1888","0xe000"]
+            # output_clusters=["0x0019"]>
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    Time.cluster_id,
+                    TuyaZBMeteringCluster.cluster_id,
+                    TuyaZBElectricalMeasurement.cluster_id,
+                    LightLink.cluster_id,
+                    0x1888,
```

Comment:
The cluster is defined in INIT so importing it and using it in´stead and also in the replacement.

Reply:
@MattWestb could you share with me the code proposal here? I'd like to learn and pick up your skills but from just comment it's hard to interpret for me unfortunately (...lack of background knowledge I guess, rookie here).

Reply:
Put it in the import like the other tuya cluster.
```
from zhaquirks.tuya import (
    TuyaZBE000Cluster,
    TuyaZBElectricalMeasurement,
    TuyaZBMeteringCluster,
    TuyaZBOnOffAttributeCluster,
)
```
and then using it like like the other is being done (dont braking the sorting in the import).
The name shall being `TuyaZB1888Cluster` if i have reading it OK.

Reply:
@MattWestb please take my feedback without any negativity or arrogance, but it's **really** hard to interpret your written English... 🙏  Frankly, I don't see why we need to sophisticate it. Considering my past experience e.g. in [pull/2252](https://github.com/zigpy/zha-device-handlers/pull/2252), and ability for community contributors to superseed this initial quirk submission, as well as following [small change code review best-practices](https://abseil.io/resources/swe-book/html/ch09.html#write_small_changes), I believe we could've just went past this.

**However**, - @erkr - do you have a second opinion on this? Can you second the reviewer's suggestion and propose (...and most importantly cross- test on your end, since there are only two of us with these cheap devices) the suggestion of:

```
from zhaquirks.tuya import (
    TuyaZB1888Cluster
    TuyaZBE000Cluster,
    TuyaZBElectricalMeasurement,
    TuyaZBMeteringCluster,
    TuyaZBOnOffAttributeCluster,
)
```
...as well as change from `0x1888,` to `TuyaZB1888Cluster.cluster_id,`?

Reply:
I've tested it and it works, resolving.